webapp/frontend/PHP/exam/scan/select-file.php

```diff
@@ -44,9 +46,34 @@
             ?>
             <label for="files" class="form-label">Wybierz plik ze skanami prac:</label>
             <input class="form-control" accept="application/pdf" type="file" id="files" name="files[]" multiple><br><br>
+
+            <label for="webdav-results" class="form-label">Lub wybierz plik z webdav:</label>
```

Comment:
@MateuszKepczynski I see that when I try to select students scans from webdav, it gets copied to answers keys.
Probably form ids should be swapped